third_party/optimized-svg-icons/styles.css

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Yep, I couldn't reference the contents of the file without making changes.

I extracted the svgs, and wrapped them with css classes.